crates/ark/src/modules/rstudio/stubs.R

```diff
@@ -16,6 +16,11 @@
     invisible(.ps.ui.navigateToFile(file, line, column))
 }
 
+#' @export
+.rs.api.askForPassword <- function(prompt = "Please enter your password") {
+    invisible(.ps.ui.askForPassword(prompt))
```

Comment:
I put `invisible()` here to at least keep it from automatically blurting out the password into the console, but note that this does store the password as a string in R, the same as `rstudioapi::askForPassword()`: <https://rstudio.github.io/rstudioapi/reference/askForPassword.html>

Reply:
I'm tempted to request that we just match the rstudioapi exactly i.e. no `invisible()` call. Even though it probably isn't ideal, it may make it seem like the call to `rstudioapi::askForPassword()` didn't actually work otherwise if you're just testing it out